### PR TITLE
(iOS) Share cookies amongst WKWebView instances

### PR DESCRIPTION
**Pull request by sundeepgupta**
Use a shared `WKProcessPool` to achieve this.

### CODE REVIEW

File: src/ios/CDVWKInAppBrowser.m [hunk 1]
```diff
@@ -19,10 +19,6 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 #import "CDVWKInAppBrowser.h"
 
-#if __has_include("CDVWKProcessPoolFactory.h")
-#import "CDVWKProcessPoolFactory.h"
-#endif
```
Comment (sundeepgupta) [inline src/ios/CDVWKInAppBrowser.m:24]: We're not using this, and when I looked it up, it's not really very helpful as the implementation only ever allocates a single instance, and then keeps reusing it.

https://github.com/apache/cordova-plugin-wkwebview-engine/blob/8735c4615a1135bf7d5c50753e44616641236664/src/ios/CDVWKProcessPoolFactory.m

File: src/ios/CDVWKInAppBrowser.m [hunk 2]
```diff
@@ -132,7 +129,11 @@ - (void)openInInAppBrowser:(NSURL*)url withOptions:(NSString*)options
         NSDate* dateFrom = [NSDate dateWithTimeIntervalSince1970:0];
         [dataStore removeDataOfTypes:[WKWebsiteDataStore allWebsiteDataTypes] modifiedSince:dateFrom completionHandler:^{
             NSLog(@"Removed all WKWebView data");
-            self.inAppBrowserViewController.webView.configuration.processPool = [[WKProcessPool alloc] init]; // create new process pool to flush all data
+
+            // Create a new process pool to flush all data.
+            WKProcessPool *newProcessPool = [[WKProcessPool alloc] init];
+            self->_processPool = newProcessPool;
+            self.inAppBrowserViewController.webView.configuration.processPool = newProcessPool;
```
Comment (sundeepgupta) [inline src/ios/CDVWKInAppBrowser.m:136]: Not sure if this code is even relevant to us as I couldn't get it to execute while playing around with the app. But I wanted to maintain the existing behaviour in any case.

File: src/ios/CDVWKInAppBrowser.m [hunk 3]
```diff
@@ -744,9 +746,7 @@ - (void)createViews
     }
     configuration.applicationNameForUserAgent = userAgent;
     configuration.userContentController = userContentController;
-#if __has_include("CDVWKProcessPoolFactory.h")
-    configuration.processPool = [[CDVWKProcessPoolFactory sharedFactory] sharedProcessPool];
-#endif
+    configuration.processPool = _processPool;
```
Comment (sundeepgupta) [inline src/ios/CDVWKInAppBrowser.m:749]: See [the comment above](https://github.com/sundeepgupta/cordova-plugin-inappbrowser/pull/1/files#r547396659)

**Review by maxkorytko — COMMENTED** [review]
The code looks good to me.

**Review by maxkorytko — APPROVED** [review]

**Pull request merged**